src/sentry/workflow_engine/handlers/condition/group_event_handlers.py

```diff
@@ -29,3 +34,45 @@ class EventSeenCountConditionHandler(DataConditionHandler[WorkflowJob]):
     def evaluate_value(job: WorkflowJob, comparison: Any) -> bool:
         event = job["event"]
         return event.group.times_seen == comparison
+
+
+@condition_handler_registry.register(Condition.EVENT_ATTRIBUTE)
+class EventAttributeConditionHandler(DataConditionHandler[WorkflowJob]):
+    @staticmethod
+    def evaluate_value(job: WorkflowJob, comparison: Any) -> bool:
```

Comment:
nit: if there's a lot happening in this function you can also break it up into several smaller ones